src/app/components/home/home.component.html

```diff
@@ -32,6 +32,15 @@ <h1>All projects in one place.</h1>
   </div>
 </div>
 
+<ng-template #testModal>
```

Comment:
Create a new component (test-dialog.component.html, test-dialog.component.scss, test-dialog.component.ts. Dialog can also be model) 
Then you can call the model in the required page with:
this.dialog.open(TestDialogComponent);
and then you don't the service

Reply:
Might be nice to check this webpage:
[popups in angular2](https://medium.com/@ladyleet/popups-modals-and-navigation-using-angular-material-2-components-in-your-angular-2-project-faf510dbcdee)
I know we're not using material design but this should work.